README.md

```diff
@@ -1,85 +1,356 @@
+          phylum_token: ${{ secrets.PHYLUM_TOKEN }}
+
+          # NOTE: These are examples. Only one `github_token` entry line is expected.
+          #
+          # Use the default `GITHUB_TOKEN` provided automatically at the start of each workflow run.
+          # This entry does not have to be specified since it is the default.
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          # Use a personal access token (PAT)
+          github_token: ${{ secrets.GITHUB_PAT }}
+
+          # NOTE: These are examples. Only one `cmd` entry line is expected.
+          #
+          # Use the defaults for all the arguments.
+          # The default behavior is to only analyze newly added dependencies against
+          # the risk domain threshold levels set at the Phylum project level.
+          # This entry does not have to be specified since it is the default.
+          cmd: phylum-ci
+          # Consider all dependencies in analysis results instead of just the newly added ones.
+          # The default is to only analyze newly added dependencies, which can be useful for
+          # existing code bases that may not meet established project risk thresholds yet,
+          # but don't want to make things worse. Specifying `--all-deps` can be useful for
+          # casting the widest net for strict adherence to Quality Assurance (QA) standards.
+          cmd: phylum-ci --all-deps
+          # Some lockfile types (e.g., Python/pip `requirements.txt`) are ambiguous in that
+          # they can be named differently and may or may not contain strict dependencies.
+          # In these cases, it is best to specify an explicit lockfile path.
+          cmd: phylum-ci --lockfile requirements-prod.txt
+          # Thresholds for the five risk domains may be set at the Phylum project level.
+          # They can be set differently for CI environments to "fail the build."
+          # NOTE: The shortened form is used here for brevity, but the long form might be more
+          #       descriptive for future readers. For instance `--vul-threshold` instead of `-u`.
+          cmd: phylum-ci -u 60 -m 60 -e 70 -c 90 -o 80
+          # Install a specific version of the Phylum CLI.
+          cmd: phylum-ci --phylum-release 3.5.0 --force-install
+          # Mix and match for your specific use case.
+          cmd: phylum-ci -u 60 -m 60 -e 70 -c 90 -o 80 --lockfile requirements-prod.txt --all-deps
+```
+
+## Continue on Error
+
+The above examples will fail the workflow when issues are found.
+If the desire is to ensure the workflow continues, even if Phylum finds issues with dependencies, then
+`continue-on-error` can be used at the [step level][1] and/or the [job level][2].
+
+[1]: https://docs.github.com/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepscontinue-on-error
+[2]: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idcontinue-on-error
```

Comment:
This seems like an odd thing to publicize. It's a feature for GitHub Actions in general and not our action specifically. And it is a less secure setup for users. I think this should be left undocumented.

Reply:
The thinking behind including this section was:

* Prepare for [GitHub Code Scanning](https://docs.github.com/en/code-security/code-scanning/automatically-scanning-your-code-for-vulnerabilities-and-errors/about-code-scanning) integration
  * This will likely require users of the action to use the `--all-deps` and `--force-analysis` options
* Preemptively head off questions regarding how to obtain this feature
  * Documentation here will save our future selves the trouble of finding an answer
  * Likely to come up "for existing code bases that may not meet established project risk thresholds yet, but still want to know the full and current state of their dependency health."

Reply:
> * Likely to come up "for existing code bases that may not meet established project risk thresholds yet"

This use case is handled by not supplying `--all-deps` and then the action only checks new dependencies and the existing state of the project doesn't matter.

> * Documentation here will save our future selves the trouble of finding an answer

True. But it also deprives us of the opportunity to encourage the customer to do something less heavy-handed.